xml/entity-decl.ent

```diff
@@ -387,6 +387,39 @@
   </para>
 </note>'>
 
+<!-- ============================================================= -->
+<!--                 Re-used Content (Multiple Books)              -->
+<!-- ============================================================= -->
+
+<!ENTITY kernel_cpu_mitigations_auto
+ '<para xmlns="http://docbook.org/ns/docbook">
+   Enables all mitigations required for your CPU model, but does
+   not protect against cross-CPU thread attacks. This setting may impact
+   performance to some degree, depending on the workload.
+  </para>'>
+
+<!ENTITY kernel_cpu_mitigations_nosmt
+  '<para xmlns="http://docbook.org/ns/docbook">
+    Provides the full set of available security mitigations. Enables all
+    mitigations required for your CPU model. In addition, it disables
+    Simultaneous Multithreading (SMT) to avoid side-channel attacks across
+    multiple CPU threads. This setting may further impact performance,
+    depending on the workload.
+    </para>'>
+
+<!ENTITY kernel_cpu_mitigations_off
+  '<para xmlns="http://docbook.org/ns/docbook">
+    Disables all mitigations. Side-channel attacks against your CPU
+    are possible, depending on the CPU model. This setting has no impact
+    on performance.
+   </para>'>
+
+<!ENTITY kernel_cpu_mitigations_manual
+  '<para xmlns="http://docbook.org/ns/docbook">
+    Does not set any mitigation level. Specify your CPU mitigations manually
+    by using the kernel command line options.
```

Comment:
Hi, in general looks good. Maybe we can mention here that if nothing is set, then kernel defaults for each cpu mitigation is used. But it is up to you if you think it is important to mention it. Also maybe mention performance impact ( impact depends on used mitigation and can vary a lot ). But I am not sure if we do not go too much into details.

Reply:
Hi Josef, the performance/security balance is mentioned in the other files (ay_bigfile.xml, grub2_yast_i.xml, inst_yast2.xml), so I think we are good on that one. And about the kernel defaults: I just noticed that I forgot to merge one more XML file yesterday [deployment_boot_parameters.xml  (SLE 15) is part of inst_yast2.xml for SLE 12]. Sorry for that, I have added the changes now. In that file, we mention: 'Each value comes with a set of specific parameters, depending on the CPU  architecture, the kernel version, and on the vulnerabilities that need  to be mitigated. Refer to the kernel documentation for details.' Would that be enough?

Reply:
yes, kernel documentation is ideal source if user want to handle it himself. So good enough

Reply:
OK, thank you!